pontoon/machinery/views.py

```diff
@@ -75,6 +76,8 @@ def concordance_search(request):
     try:
         text = request.GET["text"]
         locale = request.GET["locale"]
+        page_results_limit = int(request.GET.get("limit", 100))
+        page = int(request.GET.get("page", 1))
     except MultiValueDictKeyError as e:
```

Comment:
We should also catch a `ValueError` here in case `limit` or `page` aren't integers. Then we also don't need to catch `PageNotAnInteger` below.